src/crypto.rs

```diff
@@ -969,7 +969,7 @@ fn extract_rsa_pub_from_pkcs8(der_key: &[u8]) -> ::std::result::Result<Vec<u8>,
 
                     let n = derp::positive_integer(input)?;
                     let e = derp::positive_integer(input)?;
-                    let _ = input.skip_to_end();
+                    input.skip_to_end();
```

Comment:
Just to double check, doesn't this make clippy sad?

Reply:
https://github.com/in-toto/in-toto-rs/runs/7151973757?check_suite_focus=true#step:5:183